packages/back-end/src/api/experiments/experiments.router.ts

```diff
@@ -3,14 +3,18 @@ import { listVisualChangesets } from "../visual-changesets/listVisualChangesets"
 import { getExperimentResults } from "./getExperimentResults";
 import { getExperiment } from "./getExperiment";
 import { listExperiments } from "./listExperiments";
+import { putExperiment } from "./putExperiment";
+import { postExperiment } from "./postExperiment";
 
 const router = Router();
 
 // Experiment Endpoints
 // Mounted at /api/v1/experiments
 router.get("/", listExperiments);
+router.post("/", postExperiment);
 router.get("/:id", getExperiment);
 router.get("/:id/results", getExperimentResults);
+router.put("/:id", putExperiment);
```

Comment:
And change this to `post`

```suggestion
router.post("/:id", updateExperiment);
```